### PR TITLE
Added nginx to the rmt-server package - Card 15

### PR DESCRIPTION
**Pull request by tmuntan1**

### CODE REVIEW

File: package/rmt-server.spec [hunk 1]
```diff
@@ -40,6 +40,14 @@ Source0:        %{name}-%{version}.tar.bz2
 Source1:        rmt-server-rpmlintrc
```
Comment (lagartoflojo) [inline package/rmt-server.spec]: We should probably bump the version for this as well

**Comment by lagartoflojo** [comment]
Getting a weird error when trying to `osc build` the package.
Will check again next week:

```
[    2s] removing nis flags from /var/tmp/build-root/SLE_15-x86_64/etc/nsswitch.conf...
[    2s] now finalizing build dir...
[    3s] -----------------------------------------------------------------
[    3s] I have the following modifications for rmt-server.spec:
[    3s] -----------------------------------------------------------------
[    3s] ----- building rmt-server.spec (user abuild)
[    3s] -----------------------------------------------------------------
[    3s] -----------------------------------------------------------------
[    3s] + exec rpmbuild -ba --define '_srcdefattr (-,root,root)' --nosignature /home/abuild/rpmbuild/SOURCES/rmt-server.spec
[    3s] rpmbuild: error while loading shared libraries: librpmbuild.so.8: cannot open shared object file: No such file or directory
[    3s] 
[    3s] localhost failed "build rmt-server.spec" at Fri Feb  9 15:57:52 UTC 2018.
```

File: package/rmt-server-sync.timer [hunk 2]
```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=RMT Sync timer
+After=rmt.service
+Requires=rmt.service
+
+[Timer]
+OnCalendar=*-*-* 01:00:00
```
Comment (lagartoflojo) [inline package/rmt-server-sync.timer]: All RMTs in the same timezone will bomb our servers at the same time! Self-inflicted DDoS FTW!

Maybe we can randomize this.

Comment (digitaltom) [reply package/rmt-server-sync.timer]: 🙏 please randomize :-), or at least not at the same time as all SMTs (they also DDoS us at 1:00). 

Using something like `H H * * *` might be good.

Comment (tmuntan1) [reply package/rmt-server-sync.timer]: I put in a random delay of 3 hours

File: package/rmt-server.spec (hunk 1 above)
Comment (tmuntan1) [reply package/rmt-server.spec]: version has been bumped to "0.0.3"

**Comment by lagartoflojo** [comment]
[LGTM](http://www.lgtm.in/i/069eAehAn)

**Pull request merged**